Grafana/README.md

```diff
@@ -0,0 +1,21 @@
+
+# N8N-Getting-Started: Grafana
+This example shows how to add an **Grafana service** to your local N8N deployment using docker-compose.
+
+### Step 1. Update the docker-compose.yml
+
+```yaml
+services:
+  ...
+  grafana:
+    image: grafana/grafana
+    container_name: grafana
+    ports:
+      - 3002:3000
+    restart: unless-stopped
+    environment:
+      - GF_SECURITY_ADMIN_USER=admin
+      - GF_SECURITY_ADMIN_PASSWORD=grafana
+    volumes:
+      - ./config:/etc/grafana/provisioning/datasources
```

Comment:
It might be nice to provide this as a supplementary compose file so that the user can be instructed to run docker compose with multiple compose files instead of modifying the primary compose file. Personally, I really like this pattern for educational repos.